### DISCUSSION TITLE
Resolving IRegionMemberLifetime Version Conflict in .NET MAUI with Prism 9.0.2.71

### DISCUSSION THREAD

**Discussion by pAtMaN-bcassa** [PrismLibrary/Prism#3010] [.NET MAUI]
I'm working on a .NET MAUI project using Prism.DryIoc.Maui 9.0.2.71-pre with .net 8, and I've encountered a version conflict issue with the IRegionMemberLifetime interface. The conflict arises because this interface appears to be present in both Prism.Maui and Prism assemblies, leading to the following error:

Error CS0433: The type 'IRegionMemberLifetime' exists in both 'Prism.Maui, Version=9.0.271.34483, Culture=neutral, PublicKeyToken=null' and 'Prism, Version=9.0.271.34483, Culture=neutral, PublicKeyToken=40ee6c3a2184dc59'

Example:
public class ExampleViewModel : ViewModelBase, IRegionAware, IRegionMemberLifetime

**Comment by dansiegel** [comment]
Thanks for reporting the issue. This will be fixed in #3012. If you have the Commercial Plus License it will be available on the private NuGet feed shortly after it has been merged. Otherwise you will be able to get the fix in the next release.
